resources/kubeconfig-kubelet

```diff
@@ -3,7 +3,7 @@ kind: Config
 clusters:
 - name: local
   cluster:
-    server: ${server}
+    server: ${server_private}
```

Comment:
I think this change will affect all deployment targets (i.e. AWS).

Reply:
From what @schu and I thought about offline, we think we should copy files with breaking changes differently on lokomotive k8s and have it override the default ones that already exist on Terraform render bootkube. Though it is not a nice way to do it, but maybe we can go with it pending when we have a better way to make it more generic. What do you think?

Reply:
Yes. I would lean towards updating https://github.com/kinvolk/terraform-render-bootkube/blob/2d88bb9491f6bc9dee4bf3399229bae530df04d9/assets.tf to check if `var.api_servers_private` is an empty list and set `server_private = "${server}"` if it is. Then it can be overwritten but there's no need to change providers. Would that work?